pyproject.toml

```diff
@@ -43,6 +43,7 @@ flake8-fixme = "^1.1.1"
 pep8-naming = "^0.11.1"
 poethepoet = "^0.10.0"
 black = "^20.8b1"
+daipe-core = "1.0.0"

```

Comment:
`>=1.0`

Reply:
Better to put daipe-core to normal dependencies (non-dev) like in https://github.com/daipe-ai/databricks-bundle/blob/master/pyproject.toml#L34